### PR TITLE
Fix cloud events

### LINKED ISSUE #809: Titles in latest news and events in /cloud too large
## Summary

On the cloud landing page in the latest news and events page, the events titles are too large.

## Process

1. Visit http://www.ubuntu.com/cloud
2. Scroll down to the "The latest Ubuntu Cloud news and events" section

## Current and expected result

The events headings are h3, as in the news headings, but are larger. They should be the same size as the news column headings.

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* BEMed the classes for news and events to enforce consistency
* updated perl script that builds events to use new classes
* updated js script that builds news to use new classes

## QA

1. go to /cloud
2. update the feed to 'http://www.stmgrts.org.uk/taleo/insights_main.xml' to test
3. see that the fonts are consistent between both news and events

## Issue / Card

Fixes #809 

## Screenshots

* large screen

* 
medium screen

* 
small screen

### CODE REVIEW

File: static/css/section/_cloud.scss [hunk 1]
```diff
@@ -139,90 +139,92 @@
...
-    background-position: 0 5px;
-    background-repeat: no-repeat;
-    display: block;
-    float: left;
-    padding: 6px 20px 6px 24px;
-    width: 100%;
-  }
+    &-title {
+      font-size: 1em;
+      @media only screen and (min-width : $breakpoint-large) {
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Whitespace please

File: static/css/section/_cloud.scss [hunk 2]
```diff
@@ -139,90 +139,92 @@
...
+    &-image {
       width: 76px;
+      border-radius: 4px;
+      margin-top: 7px;
+      margin-right: 20px;
     }
-  }
 
+    &-image-wrap {
+      background-size: 78px 78px;
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Just need `78px`?

File: static/css/section/_cloud.scss [hunk 3]
```diff
@@ -139,90 +139,92 @@
...
-    background-repeat: no-repeat;
-    display: block;
-    float: left;
-    padding: 6px 20px 6px 24px;
-    width: 100%;
-  }
+    &-title {
+      font-size: 1em;
+      @media only screen and (min-width : $breakpoint-large) {
+        font-size: 1.2578125em;
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Really? What font size you going for? You could use `rem` and just base that value off `16px`

File: static/css/section/_cloud.scss [hunk 4]
```diff
@@ -139,90 +139,92 @@
...
-    background-repeat: no-repeat;
-    clear: both;
-    display: block;
-    padding: 6px 20px 6px 24px;
-  }
+    &-location {
+      margin-left: 0;
+      background-size: 20px 20px;
+      box-sizing: border-box;
+      background-image: url('https://insights.ubuntu.com/wp-content/themes/resource-centre/static/img/location.svg');
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Dont we want to load this from the asset server?

File: static/css/section/_cloud.scss [hunk 5]
```diff
@@ -139,90 +139,92 @@
...
+      background-position: 0 5px;
+      background-repeat: no-repeat;
+      display: block;
+      float: left;
+      padding: 6px 20px 6px 24px;
+      width: 100%;
+      font-size: .92857143em;
+      @media only screen and (min-width : $breakpoint-large) {
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: White space above please.

File: static/css/section/_cloud.scss [hunk 6]
```diff
@@ -139,90 +139,92 @@
...
+      background-size: 20px 20px;
+      box-sizing: border-box;
+      background-image: url('https://insights.ubuntu.com/wp-content/themes/resource-centre/static/img/location.svg');
+      background-position: 0 5px;
+      background-repeat: no-repeat;
+      display: block;
+      float: left;
+      padding: 6px 20px 6px 24px;
+      width: 100%;
+      font-size: .92857143em;
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Seems like `rem` might tidy this up.

File: static/css/section/_cloud.scss [hunk 7]
```diff
@@ -139,90 +139,92 @@
...
-    position: relative;
-    display: inline-block;
-    max-width: 100%;
-    vertical-align: middle;
-    margin-bottom: 0;
-  }
+    &-venue {
+      background-size: 20px 20px;
+      box-sizing: border-box;
+      background-image: url('https://insights.ubuntu.com/wp-content/themes/resource-centre/static/img/gps.svg');
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Asset server?

File: static/css/section/_cloud.scss [hunk 8]
```diff
@@ -139,90 +139,92 @@
...
-  .image-wrap {
-    background-size: 78px 78px;
-    position: relative;
-    display: inline-block;
-    max-width: 100%;
-    vertical-align: middle;
-    margin-bottom: 0;
-  }
+    &-venue {
+      background-size: 20px 20px;
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: There seems like a lot of duplicated styled between `-venue` and `-location`. Could you put them in a placeholder and extend them here?

File: static/css/section/_cloud.scss [hunk 9]
```diff
@@ -139,90 +139,92 @@
...
+      box-sizing: border-box;
+      background-image: url('https://insights.ubuntu.com/wp-content/themes/resource-centre/static/img/gps.svg');
+      background-position: 0 5px;
+      background-repeat: no-repeat;
+      display: block;
+      float: left;
+      padding: 6px 20px 6px 24px;
+      width: 100%;
+      font-size: .92857143em;
+      @media only screen and (min-width : $breakpoint-large) {
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Whitespace above please.

**Comment by anthonydillon** [comment]
There are some comments the scss and to follow up on are conversation. There should be a parent block class and rename `-details-title` to just `-title` as it is not within the details section.

Also I wonder if this pattern could be written in a way that it could be used for more then new events. For example, naming the component `detailed-list` or even extending the current list styles with `list--detailed`?

File: static/css/section/_cloud.scss [hunk 10]
```diff
@@ -43,6 +43,22 @@
...
+  background-size: 20px 20px;
+  box-sizing: border-box;
+  background-position: 0 5px;
+  background-repeat: no-repeat;
+  display: block;
+  float: left;
+  padding: 6px 20px 6px 24px;
+  width: 100%;
+  font-size: .92857143em;
+  @media only screen and (min-width : $breakpoint-large) {
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Whitespace above please.

File: static/css/section/_cloud.scss [hunk 11]
```diff
@@ -43,6 +43,22 @@
...
+  margin-left: 0;
+  background-size: 20px 20px;
+  box-sizing: border-box;
+  background-position: 0 5px;
+  background-repeat: no-repeat;
+  display: block;
+  float: left;
+  padding: 6px 20px 6px 24px;
+  width: 100%;
+  font-size: .92857143em;
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: What font-size are you going for here? Could you use `rem` to be easier to read?

File: static/css/section/_cloud.scss [hunk 12]
```diff
@@ -139,90 +155,61 @@
...
-    font-size: 1em;
-    @media only screen and (min-width : $breakpoint-large) {
-      font-size: .875em;
-    }
-  }
-  // events list on /cloud
-  .event-map {
-    float: left;
+  // news and events list on /cloud
+  .newsevent{
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Space before opening curly bracket.

File: static/css/section/_cloud.scss [hunk 13]
```diff
@@ -139,90 +155,61 @@
...
 
-    img {
-      margin-right: 20px;
-      width: 76px;
+    &__title {
+      font-size: 1em;
+      @media only screen and (min-width : $breakpoint-large) {
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: White space above please.

File: static/css/section/_cloud.scss [hunk 14]
```diff
@@ -139,90 +155,61 @@
...
-    background-image: url('https://insights.ubuntu.com/wp-content/themes/resource-centre/static/img/gps.svg');
-    background-position: 0 5px;
-    background-repeat: no-repeat;
-    display: block;
-    float: left;
-    padding: 6px 20px 6px 24px;
-    width: 100%;
-  }
+    &__details {
+      color: #888;
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: Use variable please.

File: static/css/section/_cloud.scss [hunk 15]
```diff
@@ -139,90 +155,61 @@
...
+      line-height: 1.2;
+      margin-bottom: 0.5em;
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: No 0 before .

File: static/css/section/_cloud.scss [hunk 16]
```diff
@@ -139,90 +155,61 @@
...
+        background-image: url('#{$asset-server}48947556-gps.svg');
+      }
 
-  .image-wrap img {
-    border-radius: 4px;
-    margin-top: 7px;
+      &-date {
+        @extend %icon;
+        background-image: url('#{$asset-server}c7d1850b-calendar.svg');
+      }
```
Comment (anthonydillon) [inline static/css/section/_cloud.scss]: :heart:

**Comment by anthonydillon** [comment]
Looking good, few comments from the tweaks. Did you settle on `newsevent` as the block name? Reference my comment before.

**Comment by pmahnke** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;32V1RfiM&#x2F;1633-bug-fix-typography-of-news-and-events-on-cloud"> [bug] fix typography of news and events on &#x2F;cloud</a>

**Comment by pmahnke** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;km2JPDeR&#x2F;1637-review-screen-images-for-issue-fix"> review screen images for issue fix</a>

**Comment by anthonydillon** [comment]
LGTM :+1:

**Pull request merged**